### PR TITLE
Color Property: Update index.md

### PR DESCRIPTION
**Pull request by arhillis**
## Color Property

You can use the `color` property to set the color of the text in an element.  

 

You can use several methods to declare a color such as: 
* By Name (Note: this only works with certain colors)
    ```css
    h1{
        color: blue;
    }
    ```

* Hexadecimal (specified as #rrggbb)

    ``` css
    h1{ 
       color:  #0000ff;
    } 
    ```
* RGB (specified as rgb(r, g, b))

   ```css
   h1{
       color: rgb(0, 0, 255);
   }
   ```
* HSL (Hue, Lightness, Saturation)
    
    ```css
    h1{
        color: hsl(240, 100%, 50%);
    }
    ```

#### More Information

* W3 Schools site on how to format [text](https://www.w3schools.com/css/css_text.asp).
* W3 Schools site on [colors](https://www.w3schools.com/colors/default.asp).
* [Color Property](https://developer.mozilla.org/en-US/docs/Web/CSS/color) on Mozilla Developer Network (MDN).
* More about [color keywords](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#Color_keywords) on MDN.
* Documentation on [w3.org](https://www.w3.org/wiki/CSS/Properties/color).

### CODE REVIEW

**Comment by svram** [comment]
You need to add the title: Color Property at the top of the file:

----		
 -title: Color Property		
 ----

File: src/pages/css/properties/color-property/index.md [hunk 1]
```diff
@@ -1,15 +1,47 @@
----
```
Comment (Bouncey) [inline src/pages/css/properties/color-property/index.md]: Please restore this frontmatter, we use this title for the UI 👍

File: src/pages/css/properties/color-property/index.md [hunk 2]
```diff
@@ -1,15 +1,47 @@
...
-<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+You can use the `color` property to set the color of the text in an element.  
 
-#### More Information:
-<!-- Please add any articles you think might be helpful to read before writing the article -->
+ 
+
+You can use several methods to declare a color such as: 
+* By Name (Note: this only works with certain colors)
+    ```
```
Comment (Bouncey) [inline src/pages/css/properties/color-property/index.md]: We can add syntax highlighting by starting the code block off stating the language i.e.

` ```css`

File: src/pages/css/properties/color-property/index.md [hunk 3]
```diff
@@ -1,15 +1,47 @@
...
+    h1{
+        color: blue;
+    }
+    ```
+
+* Hexadecimal (specified as #rrggbb)
+
+    ```
```
Comment (Bouncey) [inline src/pages/css/properties/color-property/index.md]: We can add syntax highlighting by starting the code block off stating the language i.e.

` ```css`

File: src/pages/css/properties/color-property/index.md [hunk 4]
```diff
@@ -1,15 +1,47 @@
...
+* Hexadecimal (specified as #rrggbb)
+
+    ``` 
+    h1{ 
+       color:  #0000ff;
+    } 
+    ```
+* RGB (specified as rgb(r, g, b))
+
+   ```
```
Comment (Bouncey) [inline src/pages/css/properties/color-property/index.md]: We can add syntax highlighting by starting the code block off stating the language i.e.

` ```css`

File: src/pages/css/properties/color-property/index.md [hunk 5]
```diff
@@ -1,15 +1,47 @@
...
+   h1{
+       color: rgb(0, 0, 255);
+   }
+   ```
+* HSL (Hue, Lightness, Saturation)
+    
+    ```
```
Comment (Bouncey) [inline src/pages/css/properties/color-property/index.md]: We can add syntax highlighting by starting the code block off stating the language i.e.

` ```css`

File: src/pages/css/properties/color-property/index.md [hunk 6]
```diff
@@ -1,15 +1,47 @@
...
 
-<!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
+You can use the `color` property to set the color of the text in an element.  
 
-#### More Information:
-<!-- Please add any articles you think might be helpful to read before writing the article -->
+ 
+
+You can use several methods to declare a color such as: 
+* By Name (Note: this only works with certain colors)
```
Comment (Bouncey) [inline src/pages/css/properties/color-property/index.md]: Maybe add a link to a CSS colour chart like the one available on [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#Color_keywords)

File: src/pages/css/properties/color-property/index.md (hunk 1 above)
Comment (arhillis) [reply src/pages/css/properties/color-property/index.md]: Okay. Will put it back in.

Comment (arhillis) [reply src/pages/css/properties/color-property/index.md]: Did that fix it?

Comment (Bouncey) [reply src/pages/css/properties/color-property/index.md]: @arhillis I think you have just edited you first post on this PR.

Try clicking the `files changed` tab and then editing the file there by clicking on the pencil

**Comment by arhillis** [comment]
I couldn't figure out how to add my changes to this pull request, so I created a new one. Sorry for the inconvenience. Feel free to close this one.

**Comment by Bouncey** [comment]
YES @arhillis well done! You figured it out! 

Now, if you can edit all the code blocks to have ` ```css` that would be great!

**Comment by texas2010** [comment]
thanks!

**Pull request merged**